public/app/features/alerting/unified/CloneRuleEditor.test.tsx

```diff
@@ -164,7 +164,7 @@ describe('CloneRuleEditor', function () {
       });
 
       await waitForElementToBeRemoved(ui.loadingIndicator.query());
-      await waitForElementToBeRemoved(within(ui.inputs.group.get()).getByTestId('Spinner'));
+      await waitForElementToBeRemoved(within(ui.inputs.group.get()).queryByTestId('Spinner'));
```

Comment:
why do we need this change?

Reply:
I've also seen this one being flakey on some PRs 👍

Reply:
Fixed by https://github.com/grafana/grafana/pull/73997